pkg/cluster/clusterfiles.go

```diff
@@ -143,3 +143,494 @@ for arg in "${@}"; do
 done
 exec docker-runc "${args[@]}"
 `
+
+// This matches "https://docs.projectcalico.org/v3.1/getting-started/kubernetes/installation
+//               /hosted/kubernetes-datastore/calico-networking/1.7/calico.yaml"
+// except that the ipam is overriden to use 2 ranges, 1 as expected
+// in Calico felix, and one for ipv6. This has the affect of enabling dual-stack
+// so that code in a Pod can bind to ::1 for lo.
+const CalicoNet = `
+# Calico Version v3.1.3
```

Comment:
Aha, the yaml file was taken from projectcalico, and modified to supporting IPv6.
Hmm, I'm not sure I like the way of including the whole file in the source code.
But I don't have a better idea.